packages/fiori/src/Timeline.hbs

```diff
@@ -1,4 +1,9 @@
-<div class="ui5-timeline-root" @focusin={{_onfocusin}} dir="{{effectiveDir}}">
+<div class="ui5-timeline-root"
+	@focusin={{_onfocusin}}
+	dir="{{effectiveDir}}"
+	@keyup={{_onkeyup}}
```

Comment:
we have onkeyup handler but no function relating to it. We can remove it

Reply:
Done